### PR TITLE
refactor: [M3-6307] - MUIv5 Migration - SRC > Components > CopyTooltip

### PR DESCRIPTION
**Pull request by cpathipa**
## Description 📝

- Migrate CopyTooltip component from JSS to tss-react.
- Use named export instead of default

## How to test 🧪

- Validate CopyTooltip component in storybook
- yarn test
- yarn cy:run

### CODE REVIEW

File: packages/manager/src/components/CopyTooltip/CopyTooltip.stories.mdx [hunk 1]
```diff
@@ -42,4 +40,4 @@ If horizontal space is limited, truncate the content.
   </Story>
 </Canvas>
 
-<ArgsTable story="Copy Tooltip" exclude={['className']} sort="requiredFirst"/>
+<ArgsTable story="Copy Tooltip" exclude={['className']} sort="requiredFirst" />
```
Comment (bnussman-akamai) [inline packages/manager/src/components/CopyTooltip/CopyTooltip.stories.mdx:43]: Unrelated to your changes but in my opinion, we shouldn't be hiding certain props from Storybook

```suggestion
<ArgsTable story="Copy Tooltip" sort="requiredFirst" />
```

**Review by bnussman-akamai — APPROVED** [review]

File: packages/manager/src/components/CopyTooltip/CopyTooltip.tsx
Comment (mjac0bs) [inline packages/manager/src/components/CopyTooltip/CopyTooltip.tsx:1]: Am I understanding correctly that this file had already been migrated to tss-react and we're just cleaning up the exports (and Storybook) in this PR?

**Review by mjac0bs — COMMENTED** [review]
Let's update the changelog.

Storybook page renders correctly. ✅

File: packages/manager/src/components/CopyTooltip/CopyTooltip.tsx
Comment (cpathipa) [reply packages/manager/src/components/CopyTooltip/CopyTooltip.tsx:1]: That's correct

**Review by mjac0bs — APPROVED** [review]

**Pull request merged**